browser-tests/utils/settings.ts

```diff
@@ -1,6 +1,16 @@
+import * as fs from 'fs';
+
 import * as dotenv from 'dotenv';
 
-dotenv.config({ path: '.env.local' });
+const browserTestEnvFilePath = '.env.test.local';
+
+if (fs.existsSync(browserTestEnvFilePath)) {
+  // eslint-disable-next-line no-console
+  console.info(
+    `Using "${browserTestEnvFilePath}" to populate environment variables for browser tests.`
+  );
+  dotenv.config({ path: browserTestEnvFilePath });
+}
```

Comment:
There could be an else-statement, that infos local-users, that they could use this file.